src/prototype_creep.js

```diff
@@ -117,7 +116,7 @@ Creep.prototype.handle = function() {
   try {
     if (!this.unit()) {
       this.log('Unknown role suiciding');
-      this.suicide();
+      Creep.recycleCreep(this);
```

Comment:
same as above, and we should change the log message 'Unknown role recycling creep'